index.html

```diff
@@ -9,8 +9,32 @@
 		<link rel="stylesheet" href="styles.css">
 	</head>
 
-	<body>
+	<body class="body">
+		<header>
+			<div class="header">
+					<img src="image_geometry_1.svg" alt="geometric image" class="image_header">
+					<img src="ic_menu.svg" alt="menu" class="ic_image">
+			</div>
+		</header>
 
+		<div class="headline">
+			<div class="name">
+				Akumu
+				Bavon
+			</div>
+
+			<div class="intro">
+				Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+				If you like what you see and have a project you need coded, don’t hesitate and contact me.
+			</div>
```

Comment:
Hi @Bavon101 ✋. Please use the `p` tag instead of `div` for the "intro" class. You can check [this article](https://www.freecodecamp.org/news/semantic-html5-elements/) for more information about the semantic tag